runtime/near-vm-runner/src/near_vm_runner/runner.rs

```diff
@@ -237,10 +237,22 @@ impl NearVM {
                     let key = get_contract_cache_key(code_hash, &self.config);
                     let cache_record = cache.get(&key).map_err(CacheError::ReadError)?;
                     let Some(code) = cache_record else {
+                        metrics::COMPILED_CONTRACT_CACHE_MISS
+                            .with_label_values(&[
+                                &context.shard_id.to_string(),
+                                context.apply_reason.as_ref().map_or("unknown", |r| r.as_str()),
```

Comment:
This isn't a declared label in the metric, is it?

Reply:
added label to metric.